galaxyproject/galaxy
# Frontend tool search bug fix

Fixes #14438

## How to test the changes?
(Select all options that apply)
  1. Test script included. However, if testing locally, will need to test on an environment with realistic data (for example, from usegalaxy.org or usegalaxy.eu).
  2. Then, will need to hook the realistic (static) data into the Tool Store api which produces the `tools` variable (in this `utilities.js`).
  3. From there, you can _optionally_ hook the Results api from Whoosh which produces the `results` variable (in this `utilities.js`).
  4. Referencing the included test script should provide clarity on these manual steps above.

## License
